readme.md

```diff
@@ -21,8 +22,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://ShaoLei2.github.io/layout_stars/)
+- [TEST REPORT LINK](https://ShaoLei2.github.io/layout_stars/report/html_report/)
```

Comment:
please add these links to Pull Request description

Reply:
also check you deployment since it leads to 404 page

Reply:
I have no idea why the demo link doesn't work